index.js

```diff
@@ -38,11 +69,77 @@ app.use("/api", word);
 app.use("/api", verseInfo);
 app.use("/api", tafsir);
 
+app.all("*", (req, res, next) => {
+    res.cookie("XSRF-TOKEN", req.csrfToken());
+    next();
+});
+
 app.use(express.static(path.join(__dirname, "/docs")));
 app.route("/").get((req, res) => {
     res.sendFile(path.join(__dirname + "/docs/index.html"));
 });
 
+app.get('/test', (req, res) => {
```

Comment:
let's add these endpoints under the routes folder. Can be handled in a separate pr.

Reply:
any preference for the name of the file?

Reply:
I decided testRoutes.js